cromshell

```diff
@@ -1471,6 +1488,99 @@ function fetch-all()
   return 0
 }
 
+# Get the cost for a workflow ID:
+function cost()
+{
+  which bq &>/dev/null   
+  local r=$?
+  [[ ${r} -ne 0 ]] && error "bq does not exist.  Must install the big query command-line client." && exit 8
+
+  [ ! -e ${BQ_COST_TABLE_FILE} ] && error "Big Query cost table file does not exist.  Must populate ${BQ_COST_TABLE_FILE} with big query cost table information." && exit 9
+
+  local cost_table=$(head -n1 ${BQ_COST_TABLE_FILE})
+  local id=$1
+
+  if [[ "$(uname)" == "Darwin" ]] ; then
+    local DATE_CMD=gdate
```

Comment:
Is `gdate` available by default or is it installed as part of core utils?

Reply:
It's part of coreutils.  It seems OK to require it.

Reply:
Yeah, it's fine, good that you added a check though.